package.json

```diff
@@ -37,12 +38,23 @@
     "@babel/core": "^7.7.4",
     "@babel/preset-env": "^7.7.4",
     "@babel/standalone": "^7.7.4",
-    "jest": "^24.9.0"
+    "husky": "^4.3.8",
+    "jest": "^24.9.0",
+    "lint-staged": "^10.5.4",
+    "prettier": "2.2.1"
   },
   "licenses": [
     {
       "type": "MIT",
       "url": "http://jsbin.mit-license.org"
     }
-  ]
+  ],
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "**/*.js": "prettier --write"
```

Comment:
```suggestion
    "**/*.js": "prettier --write ."
```

Reply:
Good question. So, what happens here is as follows:

Each file that matches the glob `**/*.js` gets passed into `prettier --write` as an argument e.g. `/lib/index.js` gets picked up and lint-staged will run `prettier --write /lib/index.js`.

Reply:
Ah. Very clever. Thank you, for clearing that up.